From: Christoph Hellwig <hch@lst.de>
Date: Mon, 29 Jun 2020 15:03:58 +0200
Subject: xsk: Remove a double pool->dev assignment in xp_dma_map
Patch-mainline: v5.8-rc5
Git-commit: 53937ff7bc776aac647d0b3004d7cd21861b0f78
References: bsc#1176447

->dev is already assigned at the top of the function, remove the duplicate
one at the end.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
Link: https://lore.kernel.org/bpf/20200629130359.2690853-4-hch@lst.de
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 net/xdp/xsk_buff_pool.c |    1 -
 1 file changed, 1 deletion(-)

--- a/net/xdp/xsk_buff_pool.c
+++ b/net/xdp/xsk_buff_pool.c
@@ -193,7 +193,6 @@ int xp_dma_map(struct xsk_buff_pool *poo
 	if (pool->unaligned)
 		xp_check_dma_contiguity(pool);
 
-	pool->dev = dev;
 	pool->dma_need_sync = !xp_check_cheap_dma(pool);
 	return 0;
 }
